dist/index.es.js

```diff
@@ -0,0 +1,1665 @@
+import React, { Component } from 'react';
```

Comment:
Add dist folder to git ignore. not required here

Reply:
We will require /dist until we publish this to npmjs.org